### DISCUSSION TITLE
Generator and furnaces should be able to filter input

### DISCUSSION THREAD

**Discussion by bentterp** [TechReborn/TechReborn#2518] [Ideas]
**Describe the bug**
Generator and furnaces should be able to filter input instead of importing things which they can't process.

**Steps to Reproduce**
- Create a generator with an input from a chest
- Load or into chest
- Generator pulls in ore but can't process it

**Environment (please complete the following information with the version):**
 - Minecraft: 1.6
 - Mod Loader: fabric

**Comment by modmuss50** [comment]
Yes, this is true. The reason this doesnt happen right now is that these machines dont use the same recipe system as the other machines. Id like to redo the whole filter inputs thing at someopint to allow you to specify items.

**Comment by bentterp** [comment]
It all comes down to priorities, and you probably have something more than this hobby to occupy your time :-)

If I get to make a wish, I think that the problem with cables should have a higher priority than this.
